doc/configuration.rst

```diff
@@ -143,6 +143,56 @@ You can specify the following options in module configuration.
         position = 'center'
     }
 
+.. note::
+    New in version 3.16
+
+You can specify the options in module or py3status configuration section.
+
+The following options will work on ``i3``.
+
+``align``: Specify how modules should be aligned when the ``min_width``
+is not reached. Either ``left`` (default), ``center``, or ``right``.
+``markup``: Specify how modules should be parsed.
+``min_width``: Specify a minimum width of pixels for modules.
+``separator``: Specify a separator boolean for modules.
+``separator_block_width``: Specify a separator block width for modules.
+
+The following options will work on ``i3-gaps``.
+
+``background``: Specify a background color for modules.
```

Comment:
background is supported by i3bar, so should be moved up imo

Reply:
actually i3bar supports it but not i3status modules, so we should move this under the rephrased "py3status modules" above

Reply:
I think that mean `background` should be moved up to `... will work on i3` instead.